src/test-run/commands/index.js

```diff
@@ -188,6 +188,23 @@ export class TypeTextCommand extends Assignable {
     }
 }
 
+export class NavigateToCommand extends Assignable {
+    constructor (obj) {
+        super(obj);
+
+        this.type = TYPE.navigateTo;
+        this.url  = null;
+
+        this._assignFrom(obj, true);
+    }
+
+    _getAssignableProperties () {
+        return [
+            { name: 'url', type: nonEmptyStringArgument, required: true }
```

Comment:
@inikulin do you mean we need a `NavigateToCommand` `url` validator and we should raise an error if the `url` arguement is not valid?

And what the reason to test different urls here? We just call the `hammerhead.navigateTo`function , I suppose it should be tested in `hammerhead`?